lib/stdlib/src/erl_lint.erl

```diff
@@ -112,6 +112,10 @@ value_option(Flag, Default, On, OnVal, Off, OffVal, Opts) ->
 	       on_load=[] :: [fa()],		%On-load function
 	       on_load_line=erl_anno:new(0)	%Line for on_load
                    :: erl_anno:anno(),
+               atoms=undefined                  %Declared atoms
+                   :: undefined
```

Comment:
Please do not do that.  It's fundamentally _not_ OK to mix an opaque type with a structured one because it requires that you pattern match on a structure to distinguish between them (which breaks the opacity of the opaque type).  It's probably the case that the current version of dialyzer does not disallow this, but it's quite likely that a future version will.  It's much better that the code is ready for that time.